### PR TITLE
[angular-scroll] Fix module exports

### PR DESCRIPTION
**Pull request by xemlock**
This PR makes type definitions compatible with versions >=1.0.0, by fixing invalid module exports.

angular-scroll since version 1.0.0 exports a `'duScroll'` string ([see here](https://github.com/oblador/angular-scroll/blob/v1.0.0/index.js)), which can be used directly as an AngularJS module dependency, without the need of typing manually module name e.g.:

```js
const duScroll = require('angular-scroll');

angular.module('app', [duScroll, ...])...
```

This can easily be verified in node (first line is for setting up browser environment w/ AngularJS):

```js
> require('browser-env')(); require('angular/angular'); global.angular = window.angular;
> require('angular-scroll');
'duScroll'
```

----

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: **see description above**
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## angular-scroll ([<kbd>unpkg</kbd>](https://unpkg.com/browse/angular-scroll@latest/))
was missing the following properties:
1. padStart
1. padEnd
1. trimLeft
1. trimRight
1. trimStart

<details>
<summary>as well as these 1 other properties...</summary>
<p>trimEnd</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 1493fa598137f08d633f794f5c39ecafda20a341
</p>

**Comment by xemlock** [comment]
@DangerBotOSS Sorry bot, your totally wrong. None of the functions you mentioned are used in the angular-scroll codebase.

- [padStart](https://github.com/oblador/angular-scroll/search?q=padStart)
- [padEnd](https://github.com/oblador/angular-scroll/search?q=padEnd)
- [trimLeft](https://github.com/oblador/angular-scroll/search?q=trimLeft)
- [trimRight](https://github.com/oblador/angular-scroll/search?q=trimRight)
- [trimStart](https://github.com/oblador/angular-scroll/search?q=trimStart)
- [trimEnd](https://github.com/oblador/angular-scroll/search?q=trimEnd)

All of the above links yield no results.

File: types/angular-scroll/index.d.ts [hunk 1]
```diff
@@ -1,45 +1,11 @@
-// Type definitions for angular-scroll
+// Type definitions for angular-scroll 1.0.0
```
Comment (peterblazejewicz) [inline types/angular-scroll/index.d.ts:1]:
```suggestion
// Type definitions for angular-scroll 1.0
```

File: types/angular-scroll/angular-scroll.d.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+/// <reference types="angular" />
+
+declare namespace duScroll {
+
+    /**
+     * Extends the angular.element object returned by the $document sercive with a few jQuery like functions.
```
Comment (samherrmann) [inline types/angular-scroll/angular-scroll.d.ts:6]: Typo: `sercive` => `service` (that was my mistake in the original version)

File: types/angular-scroll/index.d.ts [hunk 3]
```diff
@@ -1,45 +1,11 @@
...
-        duScrollLeft(left: number, duration: number, easing?: Function): ng.IPromise<void>;
-
-        duScrollLeftAnimated(left: number): ng.IPromise<void>;
-        duScrollLeftAnimated(left: number, duration: number, easing?: Function): ng.IPromise<void>;
-
-        duScrollTop(): number;
-        duScrollLeft(): number;
-    }
-}
+declare const _: string;
```
Comment (peterblazejewicz) [inline types/angular-scroll/index.d.ts:10]:
```suggestion
declare const _: 'duScroll';
```
and it's ready to go

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@xemlock thx!

**Review by samherrmann — APPROVED** [review]

**Comment by xemlock** [comment]
What I'm supposed to do now?

@typescript-bot merge

perhaps? 🙂

**Comment by xemlock** [comment]
@typescript-bot Ready to merge

**Comment by xemlock** [comment]
Ready to merge

**Pull request merged**